Procfile

```diff
@@ -4,6 +4,7 @@ worker_pushlog: newrelic-admin run-program celery -A treeherder worker -Q pushlo
 worker_buildapi_pending: newrelic-admin run-program celery -A treeherder worker -Q buildapi_pending --maxtasksperchild=20 --concurrency=5
 worker_buildapi_running: newrelic-admin run-program celery -A treeherder worker -Q buildapi_running --maxtasksperchild=20 --concurrency=5
 worker_buildapi_4hr: newrelic-admin run-program celery -A treeherder worker -Q buildapi_4hr --maxtasksperchild=20 --concurrency=1
+worker_pulse_jobs: newrelic-admin run-program celery -A treeherder worker -Q store_pulse_jobs --maxtasksperchild=20 --concurrency=1
```

Comment:
There currently isn't a bin scripts equivalent of this? (We just have the currently unused script for consuming pulse, not the one for handling the resultant jobs on this celery queue).

Also, at some point (to cooincide with fubar getting the `bin/run_pulse_ingestion` script to run on stage/prod), we'll need a `newrelic-admin run-program ./manage.py ingest_from_pulse` line added to Procfile too. Though Heroku will also need the relevant environment variables set before that happens I guess? (Or are we not submitting to  {heroku prototype, SCL3 stage, SCL3 prod} and instead waiting until SCL3 stage migrates to Heroku?)

If this PR is not actually adding the `newrelic-admin run-program ./manage.py ingest_from_pulse` line to Procfile, the commit message / PR title is probably better as something else, since it's not turning on ingestion as the current name implies :-)